packages/fds-components/src/components/buttons/GroupButton.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+
+export const GroupButton = (props) => {
+  const { className, active, Icon, children, ...rest } = props;
+
+  const rootClass = cx('group-btn', className, {
+    active,
+  });
+
+  return (
+    <button className={rootClass} {...rest}>
+      {Icon && <Icon size="xs" className="margin--right--half" />}
+      {children}
+    </button>
+  );
+};
+
+GroupButton.propTypes = {
+  /** Displays active button style when true */
+  active: PropTypes.bool,
```

Comment:
For boolean props in FDS, I've been using `is`/`has` conventions as an extra hint. In this case, it would be `isActive`